homeassistant/components/sensor/gtfs.py

```diff
@@ -275,6 +293,21 @@ def update(self):
             route = self._departure['route']
             trip = self._departure['trip']
 
+            if self._mode == MODE_DEPARTURE:
+                self._state = datetime.datetime.strptime(
+                    origin_stop_time["Departure Time"],
+                    TIME_FORMAT).strftime('%H:%M')
+                self._unit_of_measurement = None
+            elif self._mode == MODE_TIMER or self._mode == MODE_AUTO and \
+                    abs(minutes) >= 60:
+                hour = int(minutes / 60)
+                minute = abs(minutes) % 60
+                self._state = '{0}:{1:02d}'.format(hour, minute)
```

Comment:
It would be better to change this sensor to set an absolute utc time stamp as state and use the `timestamp` device class. This will allow the user to customize the representation in the frontend to eg minutes left to that time (departure).

Reply:
Makes sense, and it would reduce the code bloat too! This obviously will introduce another braking change however.

I'll update this PR accordingly, and will drop the documentation PR.